### ISSUE TITLE
DownloadManager crashes for some epic game installs (Kerbal Space Program)

### ISSUE THREAD

**Issue by praveenivp** [Heroic-Games-Launcher/HeroicGamesLauncher#3101]
### Describe the bug

Installing Kerbal space program from epic store crash the launcher. It looks like a game specific bug. Nevertheless, the launcher shouldn't crash.

### Add logs

```text
(01:20:30) DEBUG:   [Backend]:          {"free":52412743680,"diskSize":257547902976,"message":"48.81 GiB / 239.86 GiB","validPath":true}
(01:20:30) DEBUG:   [Backend]:          {"free":52412743680,"diskSize":257547902976,"message":"48.81 GiB / 239.86 GiB","validPath":true}
(01:20:31) INFO:    [Backend]:          Starting the Download Queue
(01:20:42) INFO:    [Backend]:          Writing config for a1e2ce30defe4a9187ebc14fc9d2bd8b
(01:20:42) INFO:    [Backend]:          Changed config: winePrefix from /home/praveen/Games/Heroic/Prefixes/Kerbal Space Program to /home/praveen/.wine
(01:20:42) INFO:    [Backend]:          Changed config: wineCrossoverBottle from  to Heroic
(01:20:42) DEBUG:   [Legendary]:        Using cached install info
(01:20:42) INFO:    [DownloadManager]:  Kerbal Space Program  was added to the download queue.
(01:20:42) INFO:    [Backend]:          Preventing machine to sleep

(heroic:13): libnotify-WARNING **: 01:20:42.929: Running in confined mode, using Portal notifications. Some features and hints won't be supported
(01:20:42) DEBUG:   [Legendary]:        Using cached install info
(01:20:42) ERROR:   [DownloadManager]:  Installation of a1e2ce30defe4a9187ebc14fc9d2bd8b failed with: [
  {
    "code": "too_small",
    "minimum": 1,
    "type": "string",
    "inclusive": true,
    "exact": false,
    "message": "String must contain at least 1 character(s)",
    "path": []
  }
]
(01:20:42) WARNING: [DownloadManager]:  Installation of a1e2ce30defe4a9187ebc14fc9d2bd8b failed!
(01:20:42) INFO:    [DownloadManager]:  a1e2ce30defe4a9187ebc14fc9d2bd8b added to download manager finished.
(01:20:42) INFO:    [DownloadManager]:  a1e2ce30defe4a9187ebc14fc9d2bd8b removed from download manager.
(01:20:42) INFO:    [Frontend]:         Refreshing legendary Library
(01:20:42) INFO:    [Legendary]:        Refreshing library...
(01:20:42) INFO:    [Legendary]:        Refreshing Epic Games...
(01:20:42) INFO:    [Legendary]:        Game list updated, got 2 games & DLCs
(01:20:43) INFO:    [Legendary]:        Running command: XDG_CONFIG_HOME=/home/praveen/.var/app/com.heroicgameslauncher.hgl/config/heroic/legendaryConfig /app/bin/heroic/resources/app.asar.unpacked/build/bin/linux/legendary list --third-party
/app/bin/heroic-run: line 6:    13 Segmentation fault      (core dumped) zypak-wrapper /app/bin/heroic/heroic "$@"

```

### Steps to reproduce

1. install kerbal space program from epic store

### Expected behavior

Display the error and continue working!

### Heroic Version

Latest Stable (Flatpak)

### System Information

CPU: 8x Intel(R) Core(TM) i5-8265U CPU @ 1.60GHz
Memory: 16.48 GB (used: 5.38 GB)
GPUs:
  GPU 0:
    Name: NVIDIA Corporation GP107M [GeForce GTX 1050 Mobile]
    IDs: D=1c8d V=10de SD=39fd SV=17aa
    Driver: nvidia
  GPU 1:
    Name: Intel Corporation WhiskeyLake-U GT2 [UHD Graphics 620]
    IDs: D=3ea0 V=8086 SD=39fd SV=17aa
    Driver: i915
OS: Freedesktop SDK 23.08 (Flatpak runtime) (linux)

The current system is not a Steam Deck
We are running inside a Flatpak container

Software Versions:
  Heroic: 2.9.2 Boa Hancock
  Legendary: 0.20.32 Dark Energy (hotfix #6)
  gogdl: 0.7.3
  Nile: 1.0.0 Jonathan Joestar

Linux user 6.5.5-arch1-1 #1 SMP PREEMPT_DYNAMIC Sat, 23 Sep 2023 22:55:13 +0000 x86_64 GNU/Linux

**Comment by gxapplications** [comment]
Same for me! Searching for 3 days a similar issue on the net...

**Comment by praveenivp** [comment]
Hey, Can you also report the game as "broken installation" in the epic store. Probably, only they can fix it. I just posted it here for the developer to see whether they want to handle this particular exception.

**Comment by gxapplications** [comment]
"broken installation" signaled, but cannot put any comment on the report...

**Comment by arielj** [comment]
I don't think they'll support a third party alternative launcher

one thing I'd recommend is trying to install the game using Legendary directly, just to know if the problem is Heroic or Legendary, then reporting it in https://github.com/derrod/legendary would make more sense if it's not heroic

**Comment by praveenivp** [comment]
I can download and play other (free) games from epic store without any issue. Does the paid/bought game behave differently?

**Comment by arielj** [comment]
Can you try with the latest 2.10.0 release? we updated Legendary

**Comment by arielj** [comment]
from a comment in another issue, this got fixed by 2.10.0
